### ISSUE TITLE
Some fields not parsed correctly, e.g, award/name

### ISSUE THREAD

**Issue by sckott** [ropensci/rcrossref#90]
```r
cr_works(query="NSF")
Source: local data frame [1 x 44]

  alternative.id                                                   container.title created  deposited
           (chr)                                                             (chr)   (chr)      (chr)
1                Journal für die reine und angewandte Mathematik (Crelles Journal)         2015-03-02
Variables not shown: DOI (chr), indexed (chr), ISBN (chr), ISSN (chr), issued (chr), member (chr),
  prefix (chr), publisher (chr), reference.count (chr), score (chr), source (chr), subject (chr), title
  (chr), type (chr), URL (chr), assertion (chr), author (chr), page (chr), funder_.award1 (chr),
  funder_.award2 (chr), funder_.award3 (chr), funder_.award4 (chr), funder_.name (chr), NA.award1 (chr),
  NA.award2 (chr), NA.award3 (chr), NA.award4 (chr), NA.name (chr), NA.award1.1 (chr), NA.award2.1
  (chr), NA.award3.1 (chr), NA.award4.1 (chr), NA.name.1 (chr), NA.award1.2 (chr), NA.award2.2 (chr),
  NA.award3.2 (chr), NA.award4.2 (chr), NA.name.2 (chr), issue (chr), volume (chr)
```

**Comment by sckott** [comment]
some field names get `NA.` before them, blargh

**Comment by sckott** [comment]
here's a call that gives problem http://api.crossref.org/works/10.1515/crelle-2013-0024

**Comment by sckott** [comment]
fixed now in d6297cb
